File: sentimental-mario/mario.py
from cs50 import get_int, get_string


def main():
    # Keep asking until the user gives a positive int
    while True:
        h = get_int("Height: ")
        if h > 0 and h < 9:
            break
    counter1 = 1
    counter2 = h-counter1
    # print empty space first and then hash
    for i in range(h):
        for j in range(h-counter1):
            print(" ", end="")
        # after each iteration, there's gonna be less empty space
        counter1 += 1
        for k in range(h-counter2):
            print("#", end="")
        # after each iteration, there's gonna be more hash space
        counter2 -= 1
        print()


main()

File: mario/mario.c
// h is hash, s is space

#include <cs50.h>
#include <stdio.h>

int get_positive_int(string prompt);

int main(void)
{
    int n = get_positive_int("Choose height: ");
    int x = n-1;
    
    for (int i = 0; i < n; i = i+1)
        
    {
        for (int s = 0; s < x ; s++)
        {
            printf(" ");
            
        }
        
        for (int h = 0; h<i+1; h++)
        {
            printf("#");
        }
       
        printf("\n");
        x = x-1;
      
    }
}

int get_positive_int(string prompt)
{
    int n;
    do
    {
        n = get_int("%s", prompt);
    }
    while (n < 1 || n > 8);
    return n;
}
